esmvaltool/cmorizers/obs/cmorize_obs_merra2.py

```diff
@@ -123,13 +121,20 @@ def _extract_variable(in_files, var, cfg, out_dir):
     nlon = len(cube.coord('longitude').points)
     cube.data = da.roll(cube.core_data(), int(nlon / 2), axis=-1)
 
+    # Fix time
+    if var['mip']=='Lmon':
+        cube = _fix_time_monthly(cube)
+    elif var['mip']=='day':
+        cube = _fix_time_daily(cube)
+    else:
+        raise NotImplementedError
```

Comment:
maybe verbose here a bit what's not implemented eg "Data frequency should be either monthly or daily"